src/style.css

```diff
@@ -1 +1,103 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("/src/fonts/Avenir-Book.ttf") format("truetype"),
+    url("/src/fonts/Avenir-Heavy.ttf") format("truetype"),
+    url("/src/fonts/Avenir.ttc") format("truetype");
+  font-weight: 300px;
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+}
+
+.form-big, .form-small {
+  margin: 20px 8px 0;
+}
+
+/* .form-big::before, .form-small::before {
+  content:"";
+  height: 20px;
+  display: block;
+} */
+
+.fieldset {
+  padding: 0;
+  margin: 0;
+  border: none;
+  position: relative;
+}
+
+.i-search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  color:#3d4e61;
+  font-family: Avenir, sans-serif;
```

Comment:
Set this property to html and here only inherit it

Reply:
Done